### DISCUSSION TITLE
500m image CC dataset?

### DISCUSSION THREAD

**Discussion by TruthSearcher** [ml6team/fondant#687] [Q&A] [answered]
500m image CC dataset expected release?

**Comment by janvanlooyml6** [comment, marked as answer]
Hi Truthsearcher, no fixed date yet. More news will follow.

**Reply by nunbuzor** [reply]
> Hi Truthsearcher, no fixed date yet. More news will follow.

any update on this? or is this abandoned?

**Reply by nunbuzor** [reply]
@GeorgesLorre
